### ISSUE TITLE
Multilingual option - full support for Polylang, WPML & others - Templates, Reusables

### ISSUE THREAD

**Issue by tottello** [soflyy/oxygen-bugs-and-features#1408]
**Describe the feature you'd like to see included in Oxygen.**

I would like to translate my websites easily, without 'hacks'.

**What are the use cases for this feature?**
Any multilingual website

**Examples of this feature or functionality.**

For me the biggest issue is with Reusables. In Polylang I can translate Reusables but they will not be replaced automatically. If I translate them I have to put them again in every page they occur. 

Another is adding new templates (translated) and this still require change used 'template' in page edit section.

**Comment by Anticosti** [comment]
And with WP Multilang (a bloat free solution).

**Comment by Anticosti** [comment]
And with WP Multilang (a bloat free solution).

**Comment by dmorawetz** [comment]
[TranslatePress](https://wordpress.org/plugins/translatepress-multilingual/) works great with Oxygen without any hacks. It's user friendly, so it can also be used by clients. Please add it do documentation.

**Comment by lllopo** [comment]
> [TranslatePress](https://wordpress.org/plugins/translatepress-multilingual/) works great with Oxygen without any hacks. It's user friendly, so it can also be used by clients. Please add it do documentation.

True, but unfortunately TranslatePress doesn't play well with other plugins, like SearchWP for example ... so it is still nightmare to build a decent multilingual site. WPML is de facto standard for multilingual Wordpress and I think Oxygen should provide good support for it. For now, I quit using Oxygen, although I love it's feature set and paid for it ... but currently it remains a nice concept, which is simply useless in real life.

**Comment by olegyurist** [comment]
It is very important to have a multilingual site, please add support

**Comment by kartzke** [comment]
+1 for WPML support!

**Comment by netrunner7** [comment]
+1 for Polylang

**Comment by Jonas030tango** [comment]
It's kind of shocking, that new features get rolled out, while such basic things as a solid multi-language support is still not implemented...

**Comment by olehkoh13** [comment]
+1 for WPML

**Comment by bdonner** [comment]
Is the case really such still now August 2023 that there is no smoothly working way to use Oxygen on multi-lingual sites? We have a medium-sized site that we created using Oxygen and WPML and we have huge problems with translations disappearing and behaving weirdly and I find no reports of for instance Reusables being 100% compatible with any multi-lingual system for WP. Is the only way to go to trash Oxygen builder alltogether?

**Comment by swinggraphics** [comment]
"Template translation doesn't work for logged out users. As such, you will need to use one template, duplicate your content, translate it within the same template and then use Conditions to show/hide the content based on the language string in the URL." -[From the docs](https://oxygenbuilder.com/documentation/other/multilingual/)

…Really?? Surely it is possible to tell Oxygen to use a different template depending on the language, even if I have to code the logic myself, right? What is the filter to alter the template Oxygen chooses?

**Comment by daniel-pixeladas** [comment]
Hello,
is the "full support" still in your plans? We have a lot of problems translating templates.

**Comment by webitup-company** [comment]
+1 for WPML
